### PR TITLE
my first python app

### LINKED ISSUE #1: add web server using python

### PR DESCRIPTION
**Pull request by kohlideepti**

### CODE REVIEW

File: src/server.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+main(){
```
Comment (brainupgrade-in) [inline src/server.py:2]: add method comments

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
check comments and act

File: src/server.py (hunk 1 above)
Comment (kohlideepti) [reply src/server.py:2]: done

**Review by brainupgrade-in — APPROVED** [review]
ok

**Pull request merged**